metric-schema-java/src/integrationInput/java/com/palantir/test/MyNamespaceMetrics.java

```diff
@@ -20,6 +20,13 @@ public final class MyNamespaceMetrics {
     private static final String LIBRARY_VERSION =
             Objects.requireNonNullElse(MyNamespaceMetrics.class.getPackage().getImplementationVersion(), "unknown");
 
+    private static final MetricName workerUtilizationMetricName = MetricName.builder()
```

Comment:
I'm using camel case for all fields, even those that are static, to keep the implementation simpler. While not the most idiomatic Java code, I imagine it's acceptable here.

Reply:
Sounds good to me, worth a comment in the codegen along these lines (if there isn’t already, I haven’t read through the change yet, will review tomorrow).

Reply:
I'm supportive of this naming as these aren't compile time constants.